### ISSUE TITLE
Scrolling marquee on website does not respect prefers-reduced-motion accessibility flag

### ISSUE THREAD

**Issue by gregexp** [kamranahmedse/driver.js#378]
The scrolling/animated text banner on the home page of driverjs.com should be disabled when visitors have the `prefers-reduced-motion` accessibility flag set on their operating system or browser.

See https://www.smashingmagazine.com/2021/10/respecting-users-motion-preferences/

**Comment by kamranahmedse** [comment]
Thank you for opening the issue. Since this is a minor issue and will require some design effort, I am going to close this one for now and put it in the backlog as a low priority item. But feel free to submit a pull request if you want.
